modules/javafx.graphics/src/main/java/javafx/scene/LightBase.java

```diff
@@ -56,33 +56,110 @@
 import com.sun.javafx.logging.PlatformLogger;
 
 /**
- * The {@code LightBase} class provides definitions of common properties for
- * objects that represent a form of light source. These properties
- * include:
+ * The {@code LightBase} class is the base class for all objects that represent a form of light source. All light types
+ * share the following common properties that are defined in this class:
  * <ul>
- * <li>{@code color} - the color of the light source</li>
- * <li>{@code scope} - a list of nodes the light source affects</li>
- * <li>{@code exlusionScope} - a list of nodes the light source does not affect</li>
+ *   <li>{@code color} - the color of the light source</li>
+ *   <li>{@code scope} - a list of nodes the light source affects</li>
+ *   <li>{@code exlusionScope} - a list of nodes the light source does not affect</li>
  * </ul>
  *
+ * In addition to these, each light type supports a different set of properties, summarized in the following table:
+ *
+ * <table border="1">
+ *   <caption>Summary of Light types by attributes</caption>
+ *   <tr>
+ *     <th scope="col">Light type</th>
+ *     <th scope="col">Rotation and Direction</th>
+ *     <th scope="col">Location and Attenuation</th>
+ *   </tr>
+ *   <tr>
+ *     <td>{@link AmbientLight}</td>
```

Comment:
In addition to `scope="col"` for the first row, the first column in each other row needs a `scope="row:` (and should use a `<th>` element for that column). For example, see [javafx/scene/media/package.html#L92](https://github.com/openjdk/jfx/blob/3f62718b0ce00c1cab7f23cbd67c26887b4f50ad/modules/javafx.media/src/main/java/javafx/scene/media/package.html#L92).